app/lib/account_session.rb

```diff
@@ -76,14 +76,30 @@ def to_hash
 
   def get_attributes(attribute_names)
     local = attribute_names.select { |name| user_attributes.stored_locally? name }
-    remote = attribute_names.reject { |name| user_attributes.stored_locally? name }
+    remote = attribute_names.reject { |name| user_attributes.stored_locally? name }.reject { |name| user_attributes.cached_locally? name }
```

Comment:
is rejecting these in sequence important for the result?

Otherwise could do?

`.reject { |name| user_attributes.stored_locally? name && user_attributes.cached_locally? name }`

no?

Reply:
Ok, I'll merge these.